comdb2/cdb2.py

```diff
@@ -264,12 +264,20 @@ def __init__(self, database_name, tier="default", flags=0, tz='UTC',
             self._hndl.execute("set timezone %s" % tz, {})
         self._cursor = iter([])
 
-    def close(self):
+    def close(self, consume_current_event=True):
```

Comment:
Wondering on the naming: Should this be `ack_current_event`? Feels like it is already being consumed (and handled, isn't it?).

I get the perception of if an exception is raised.. etc. But what if the exception happens at some logging after it has been "what users see as consumed". Maybe even "auto_ack_current_event"? The weird thing is that users cannot manually ack though.

I don't have a great name though, if you think consume_current_event is the best we can provide, OK with me, but feels it might not be explicit enough.

Reply:
> Perhaps we should seek out some other opinions...

+1